src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -38,7 +38,7 @@ public class RecapitiDestinatarioPage extends BasePage {
     @FindBy(id = "cancelContact-default")
     WebElement eliminaPECButton;
 
-    @FindBy(xpath = "//div/h2[contains(text(),'Grazie!')]/following-sibling::div//button")
+        @FindBy(xpath = "//div/h2[contains(text(),'Grazie!')]/following-sibling::div//button")
```

Comment:
Check https://github.com/pagopa/pn-frontend-e2e/pull/130

Reply:
Take the id element or mix both with xpath and id

Reply:
@ImJimMoriarty this element was not added by me, it is a space issue which I deleted. I can not change it now because it wil take a lot of time to replicate the issue and find the webelement